openfisca_core/periods.py

```diff
@@ -403,8 +403,13 @@ def intersection(self, start, stop):
             return self.__class__((
                 'month',
                 intersection_start,
-                ((intersection_stop.year - intersection_start.year) * 12 + intersection_stop.month -
-                    intersection_start.month + 1),
+                (
+                    (intersection_stop.year - intersection_start.year)
+                    * 12
```

Comment:
Not a fan of this `* 12` alone, I find it less easy to understand than `(intersection_stop.year - intersection_start.year) * 12`. 
Following @Morendil suggestion of suggesting a commit rather than requesting changes, I'm committing a small change, let me know if the new style doesn't work for you :)